store/store.go

```diff
@@ -562,12 +562,16 @@ func (s *SnapUbuntuStoreRepository) Updates(installed []*CurrentSnap, auther Aut
 
 	currentSnaps := make([]currentSnapJson, len(installed))
 	for i, cs := range installed {
+		revision := cs.Revision.N
+		if cs.Revision.Local() {
```

Comment:
btw the reasoning for this seems to be, hide it from the store if it's not a store revision, not sure we will invent more strange revision variants, but seems safer